### PR TITLE
doc/rados: fix typo

### PR DESCRIPTION
**Pull request by Rjerk**
Signed-off-by: Liu Lan <liulan@umcloud.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
some of these issues have been addressed by #30583 , could you drop them?

**Comment by Rjerk** [comment]
@tchaikov done

File: doc/radosgw/notifications.rst [hunk 1]
```diff
@@ -17,27 +17,27 @@ user can only manage its own topics, and can only associate them with buckets it
...
-Same counters are shared between the pubsub sync module and the bucket notification mechanism.
+The same counters are shared between the pubsub sync module and the bucket notification mechanism.
 
-- ``pubsub_event_triggered``: running counter of events with at lease one topic associated with them
+- ``pubsub_event_triggered``: running counter of events with at least one topic associated with them
 - ``pubsub_event_lost``: running counter of events that had topics associated with them but that were not pushed to any of the endpoints
 - ``pubsub_push_ok``: running counter, for all notifications, of events successfully pushed to their endpoint
 - ``pubsub_push_fail``: running counter, for all notifications, of events failed to be pushed to their endpoint
-- ``pubsub_push_pending``: gauge value of events pushed to an endpoint but not acked or nacked yet
+- ``pubsub_push_pending``: gauge value of events pushed to an endpoint but not acked or acked yet
```
Comment (tchaikov) [inline doc/radosgw/notifications.rst]: this is not a type. nack means negative ack.

Comment (Rjerk) [reply doc/radosgw/notifications.rst]: changed back

**Pull request merged**